saleor/graphql/shop/utils.py

```diff
@@ -0,0 +1,22 @@
+from typing import Optional
+
+from django_countries import countries
+
+from ...shipping.models import ShippingZone
+
+
+def get_countries_codes_list(in_shipping_zones: Optional[bool] = None):
```

Comment:
I am not sure `Optional[bool]` is good approach. I think if we want to pass `bool` then it should be True/False by default.

Reply:
I added better description. We want to be able to not pass this argument at all (as we were up till now) - which will cause all countries codes to be returned.